data/map.txt

```diff
@@ -29143,6 +29131,11 @@ planet Vatican
 	security 0
 	government Uninhabited
 
+planet Veliante
+	attributes ka'het uninhabited
+	landscape land/fog5
+	description `Veliante may have been a livable place once, but it is not like that anymore. Part of its southern hemisphere is filled with deadly levels of radiations, and even on the rest of the planet it would be safer not to remain on the surface without a suit for too long. While the animal life on the planet seems to have disappeared almost completely, plants and trees cover the surface in colorful patches of massive dimensions.`
```

Comment:
```suggestion
	description `Veliante may have been inhabitable once, but it is not anymore. Part of its southern hemisphere is filled with deadly levels of radiation, and even on the rest of the planet it would be safer not to remain on the surface without adequate protection for too long. While the animal life on the planet seems to have disappeared almost completely, plants and trees cover the surface in colorful patches of massive dimensions.`
```